src/ContainerWithTooltip.tsx

```diff
@@ -19,28 +19,33 @@ const ContainerWithTooltip = ({
   onHover: (id: string | null) => void
   isHovered: boolean
 }) => {
-  const [mousePosition, setMousePosition] = useState<[number, number, number]>([0, 0, 0])
+  const [mousePosition, setMousePosition] = useState<[number, number, number]>([
+    0, 0, 0,
+  ])
   const { camera } = useThree()
   const mouseRef = useRef(new THREE.Vector2())
-  
+
   // Update tooltip position on every frame when hovered
   useFrame(() => {
     if (isHovered) {
       // Project the stored mouse coordinates into 3D space
-      const vector = new THREE.Vector3(mouseRef.current.x, mouseRef.current.y, 0.5)
+      const vector = new THREE.Vector3(
+        mouseRef.current.x,
+        mouseRef.current.y,
+        0.5,
+      )
       vector.unproject(camera)
       setMousePosition([vector.x, vector.y, vector.z])
     }
   })
 
   const groupProps: GroupProps = {
-    position,
```

Comment:
This is the  actual fix and the rest is formatting?

Reply:
ya but the mousePosition is calcultaed wrong Im going to open an issue for it.

![image](https://github.com/user-attachments/assets/fd32b149-8600-4ab1-8b89-20e107d52ac9)